lib/p2p/Pool.ts

```diff
@@ -1,8 +1,12 @@
+import assert from 'assert';
 import net, { Server, Socket } from 'net';
 import { EventEmitter } from 'events';
 import Peer from './Peer';
 import Hosts from './Hosts';
 import NetAddress from './SocketAddress';
+import Packet from './packets/Packet';
```

Comment:
Can this be imported from `./packets` below as `{ Packet }`?

Reply:
yep, forgot to update that